src/slices/RedeemThunk.ts

```diff
@@ -48,10 +48,8 @@ export const redeemBalance = createAsyncThunk(
     } catch (e: unknown) {
       uaData.approved = false;
       const rpcError = e as IJsonRPCError;
-      if (rpcError.code === -32603 && rpcError.message.indexOf("ds-math-sub-underflow") >= 0) {
```

Comment:
Why remove this error? It's different to the "no redeemable balance" one.

Reply:
Think it's only relevant for giving. Since redeem automatically redeems the full available balance I don't think there's a risk of trying to give or redeem more than the available balance